src/comm/monitor/monitor-process.js

```diff
@@ -105,6 +105,7 @@ function getUsage(pids, type) {
         });
 
         Promise.all(promises).then((stats) => {
+            console.log(stats);
```

Comment:
This logs the CPU and memory stats for every progress update (maybe a leftover debug log?)